.huskyrc.yml

```diff
@@ -0,0 +1,3 @@
+hooks:
+  pre-commit: lint-staged && ./scripts/validate-ci-config.sh
```

Comment:
Making this a bash script means you will not be able to commit from windows. Can it be converted to a node script instead?

Reply:
Good call. I'll convert it into a node script, but the validator uses a CLI tool that isn't available on windows, so what I'll do is check if the command exists and the platform is windows before running the validator.